scripts/nayduck.py

```diff
@@ -214,6 +220,9 @@ def run_locally(tests):
         else:
             print(f'Unrecognised test category ‘{fields[0]}’', file=sys.stderr)
             continue
+        if args.dry_run:
+            print(" ".join(cmd))
```

Comment:
This needs quoting through `shlex.quote`. Also, probably inclusion of `cd` as in something like `print('( cd {} && {} )'.format(shleq.quote(str(cwd)), ' '.join(shlex.quote(str(arg)) for arg in cmd)))`.

Reply:
@mina86 Thanks, fixed.